src/examples/table/TableDraggable.tsx

```diff
@@ -188,7 +188,7 @@ const Example = () => {
       <Table>
         <Head>
           <HeaderRow>
-            <HeaderCell isMinimum />
+            <HeaderCell isMinimum hidden />
```

Comment:
To hide this accessibly, won't we need some text for the Cell here?

Reply:
Yes, I'd recommend using "Draggable grip" as the content.

Reply:
Good call.